src/sass/components/_feature-box.scss

```diff
@@ -61,3 +63,15 @@
     }
   }
 }
+
+@media screen and (min-width: 320px) and (max-width: 766px) {
```

Comment:
skoro niżej jest `768` to tu powinno być `767`, żeby style zmieniały się płynnie

Reply:
Moje niedopatrzenie, poprawię :)